packages/salesforcedx-vscode-core/src/conflict/persistentStorageService.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { FileProperties } from '@salesforce/source-deploy-retrieve/lib/src/client/types';
+import {
+  ExtensionContext,
+  Memento
+} from 'vscode';
+
+interface ConflictFileProperties {
+  lastModifiedDate: string;
```

Comment:
Currently only storing lastModifiedDate because it's the only thing needed to detect a conflict. Could easily add more fields but I don't think they're necessary